apps/Feedreader.md

```diff
@@ -28,11 +30,11 @@ screenshots:
 installation:
   - system: elementary
     info:
-    - version: 0.3
+    - version: 1.4.3
```

Comment:
this field refers to the OS version, not the app version (which would change to quickly to keep track of)